Overview.src.html

```diff
@@ -336,13 +336,23 @@ <h4 id=terminology-headers>Headers</h4>
 <hr>
 
 <p>A <dfn>simple header</dfn> is a <span title=concept-header>header</span> whose
-<span title=concept-header-name>name</span> is `<code title>Accept</code>`,
-`<code title>Accept-Language</code>`, or `<code title>Content-Language</code>`, or whose
-<span title=concept-header-name>name</span> is `<code title>Content-Type</code>` and
-<span title=concept-header-value>value</span>, <span title=concept-header-parse>once parsed</span>,
-has a MIME type (ignoring parameters) that is
-`<code title>application/x-www-form-urlencoded</code>`, `<code title>multipart/form-data</code>`, or
-`<code title>text/plain</code>`.
+<span title=concept-header-name>name</span> is one of
+
+<ul class=brief>
+  <li>`<code title>Accept</code>`
+  <li>`<code title>Accept-Language</code>`
+  <li>`<code title>Content-Language</code>`
+  <li>`<code title>Content-Type</code>` and whose <span title=concept-header-value>value</span>,
+  <span title=concept-header-parse>once parsed</span>, has a MIME type (ignoring parameters)
+  that is `<code title>application/x-www-form-urlencoded</code>`,
+  `<code title>multipart/form-data</code>`, or `<code title>text/plain</code>`
+  <li>`<code title><a href=http://httpwg.org/http-extensions/client-hints.html#the-dpr-client-hint>DPR</a></code>`,
+  `<code title><a href=http://httpwg.org/http-extensions/client-hints.html#the-downlink-client-hint>Downlink</a></code>`,
+  `<code title><a href=http://httpwg.org/http-extensions/client-hints.html#the-save-data-hint>Save-Data</a></code>`,
+  `<code title><a href=http://httpwg.org/http-extensions/client-hints.html#the-viewport-width-client-hint>Viewport-Width</a></code>`, or
+  `<code title><a href=http://httpwg.org/http-extensions/client-hints.html#the-width-client-hint>Width</a></code>`, and whose <span title=concept-header-value>value</span>,
+  <span title=concept-header-parse>once parsed</span>, is not a failure.
+</ul>
```

Comment:
I think it might be better to have `<p>or whose name is one of </p> <ul class=brief> ... </ul> <p>and value, once parsed, is not failure.</p>` after the `Content-Type` entry. It seems a little weird to have a list of header names and then have a list of header names inside that list.